File: bee1036.js
const input = require("fs").readFileSync("stdin", "utf-8")
const lines = input.split(" ")

let A = parseFloat(lines.shift())
let B = parseFloat(lines.shift())
let C = parseFloat(lines.shift())

let x = parseFloat((B * B) - (4 * A * C))

if(x < 0) {
    console.log("Impossivel calcular")
} else if(A == 0) {
    console.log("Impossivel calcular")
}else {
    let R1 = parseFloat((-B + Math.sqrt(x)) / (2 * A))
    console.log("R1: " + R1.toFixed(5))
    let R2 = parseFloat((-B - Math.sqrt(x)) / (2 * A))
    console.log("R2: " + R2.toFixed(5))
}

File: python/bee1036.py
# -*- coding: utf-8 -*-

valores = [float(x) for x in input().split(" ")]
a = valores[0]
b = valores[1]
c = valores[2]

delta = (b ** 2) - (4*a*c)

if delta < 0:
    print("Impossivel calcular")
elif a == 0:
    print("Impossivel calcular")
else:
    x1 = (-b + delta ** (0.5)) / (2*a)
    x2 = (-b - delta ** (0.5)) / (2*a)
    print(f"R1 = {x1:.5f}")
    print(f"R2 = {x2:.5f}")
